dashboard/src/components/users/UsersTable.vue

```diff
@@ -60,17 +60,26 @@ limitations under the License.
           </b-button-group>
         </template>
         <template v-slot:cell(totalPoints)="data">
-          <div class="row" :data-cy="`usr_progress-${data.item.userId}`">
-            <div class="col-auto">
-              <span class="font-weight-bold text-primary" data-cy="progressPercent">{{ calcPercent(data.value) }}%</span>
+          <div :data-cy="`usr_progress-${data.item.userId}`">
+            <div class="row">
+              <div class="col-auto">
+                <span class="font-weight-bold text-primary" data-cy="progressPercent">{{ calcPercent(data.value) }}%</span>
+              </div>
+              <div class="col text-right">
+                <span class="text-primary font-weight-bold" data-cy="progressCurrentPoints">{{ data.value | number }}</span> / <span class="font-italic" data-cy="progressTotalPoints">{{ totalPoints | number }}</span>
```

Comment:
The xx / xxxxx points could use a better aria-label for screenreader users

Reply:
:+1: